api/v1beta1/common_types.go

```diff
@@ -53,6 +53,10 @@ const (
 	//   will not be completed until the annotation is removed and all MachineDeployments are upgraded.
 	ClusterTopologyDeferUpgradeAnnotation = "topology.cluster.x-k8s.io/defer-upgrade"
 
+	// ClusterTopologyUpgradeConcurrencyAnnotation can be set on a Classy Cluster to define the maximum concurrency while
+	// upgrading MachineDeployments of a Classy Cluster.
+	ClusterTopologyUpgradeConcurrencyAnnotation = "topology.cluster.x-k8s.io/upgrade-concurrency"
```

Comment:
Please also update: https://cluster-api.sigs.k8s.io/reference/labels_and_annotations.html

Reply:
Resolving as it's a duplicate of https://github.com/kubernetes-sigs/cluster-api/pull/8432#discussion_r1164046633